src/css/components/panel.scss

```diff
@@ -157,3 +156,23 @@ h4 + .panel {
   font-weight: 600;
   padding-bottom: $grid-1;
 }
+
+.panel-routes,
+.panel-services {
+
+  h1 {
+    font-weight: 400;
+  }
+
+  h3 {
+    font-size: 0.75rem;
+
+    a {
+      color: #056dd4;
```

Comment:
What's this color from? I think all our colors should use variables from `override_vars.scss`. So this would be `color: $color-primary` or something.

I also question again why we can't just use the default styling of links.